app/ui/launcher/add.dialog.c

```diff
@@ -103,7 +101,7 @@ static void dialog_cb(lv_event_t *event) {
     uint16_t btn = lv_msgbox_get_active_btn(dialog);
     if (btn == 1) {
         const char *address = lv_textarea_get_text(controller->input);
-        if (!validate_ip4(address))return;
+        if (!address)return;
```

Comment:
Use `strlen`

Reply:
We talked about only checking the first character in the string